schema/data-auth.json

```diff
@@ -78,4 +78,7 @@
     ["Stop Broadcast","AUTH_STOPBROADCAST", true],
     ["Add Quotes","AUTH_ADD_QUOTES", true],
     ["Upload Music - Manual","AUTH_UPLOADMUSICMANUAL", true]
+    ["User can tracklist own/current show","AUTH_TRACKLIST_OWN", true],
+    ["User can tracklist for any show/timeslot","AUTH_TRACKLIST_ALL", true]
```

Comment:
Also is it just me, or is the permission naming inconsistent? None of the others have the `User can` prefix

Reply:
Fair